docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.13-alpine AS builder
+ARG SVC
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/mainflux/mainflux
+COPY . .
+RUN apk update \
+    && apk add make\
+    && make $SVC \
+    && mv build/mainflux-$SVC /exe
+
+FROM scratch
+# Certificates are needed so that mailing util can work.
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
Why is this needed? It should be volume-mounted.

Reply:
we have the same on core
mounted from where ?

Reply:
Use this generic Makefile: https://github.com/mainflux/mainflux/blob/master/tools/mqtt-bench/Makefile

It is the same over here: https://github.com/mainflux/mainflux/blob/master/tools/provision/Makefile

It's simple and reusable.

Reply:
this is dockerfile, dont get it

Reply:
No need to copy certificates into the Dockerfile.

Certificates will be broght into container via volumes defined in docker-compose.